sjb/config/test_cases/ami_build_origin_int_rhel_fork.yml

```diff
@@ -0,0 +1,206 @@
+---
+timer: 'H H * * *'
+parameters: []
+provision:
+  os: "rhel"
+  stage: "bare"
+  provider: "aws"
+actions:
+  - type: "host_script"
+    title: "install CI user account"
+    script: |-
+      oct prepare user
```

Comment:
You'll want to pick up my hack from https://github.com/openshift/aos-cd-jobs/pull/219